.github/workflows/auto-build-rpm.yml

```diff
@@ -29,6 +29,16 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
 
+      - uses: docker/setup-buildx-action@v1
+
+      - uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-apisixdashboard-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-apisixdashboard-
+            ${{ runner.os }}-buildx-
```

Comment:
Do we need to provide the restore keys? If so it will use the old cache.
Will this cause any problem?

Reply:
@starsz we need restore keys, or every new commit will cache miss. The caches are docker layers, and docker uses it only for layers that are exactly the same. so that's ok to use the old cache

Reply:
@starsz currently the apisix-build-tools only cache the fpm image for building, which may be considered as consistent. So I guess that's way @qsliu2017 use `restore-keys` here. While I must admit that, if there're other tasks for building images here, it definitely would cause stale cache problems.

@qsliu2017 What do you think? Please let us know your explanations. Thanks.

Reply:
for instance, we change a docker layer in a commit. this action will restore the cache of the last, docker computes the hash of all layers, and find caches of the same hash value(all unchanged layers). the only changed one will cache miss, and be rebuilt by docker

Reply:
@qsliu2017 thank you for the explanations. It's really helpful.

Reply:
@qsliu2017 Thank you for the explanations. I got it.